### PR TITLE
add test

### PR DESCRIPTION
**Pull request by mTeeeur**

### CODE REVIEW

**Review by askomar — CHANGES_REQUESTED** [review]
Nice work) Ty

File: main.test.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ describe('main success', () => {
         expect(sum([1, 2])).toBe(3)
     })
 
+    it('should return -1', () => {
+        expect(sum([1, -2])).toBe(-1)
+    })
```
Comment (askomar) [inline main.test.js:10]: We must not add ;?)

**Review by askomar — CHANGES_REQUESTED** [review]

File: main.test.js (hunk 1 above)
Comment (mTeeeur) [reply main.test.js:10]: We must but I am Gopher and i hate ; =)

**Review by askomar — APPROVED** [review]
Nice

**Pull request merged**